### PR TITLE
Justin kobylarz patch 1

### PR DESCRIPTION
**Pull request by JustinKobylarz**
Sorry, I have no idea if i did this correctly... (1st Timer...)

### CODE REVIEW

**Comment by pryrt** [comment]
> Sorry, I have no idea if i did this correctly...

1. You should have _edited_ the existing `udl-list.md` instead of creating the new `NEWudl-list`

2. You should have _edited_ the existing `udl-list.json` instead of creating the new `NEWudl-list.json`

3. the `udl-list.json` will need a comma between the blocks, just like there's a comma between all the other blocks

4. You did not commit ./UDLs/UniBASIC.xml, so the User Defined Language definition that you're trying to add cannot be added.  You need to include that file in your PR.

5. The name you chose for the file in the `NEWudl-list.md` file (`UniBASIC.xml`) does not match the id-name you used in the `NEWudl-list.json` file (`UniBASIC_byJustinKobylarz.xml`).  The only difference between those two fields _must_ be just the `.xml` file extension.  And when you include the file that was missing in point#3, it needs to match the full name.  

To sum up: 
* The file should be called `./UDLs/UniBASIC_byJustinKobylarz.xml`, and needs to be included in your PR
* The `udl-list.md` should reference `./UDLs/UniBASIC_byJustinKobylarz.xml`
* The `udl-list.json` should reference `"id-name": "UniBASIC_byJustinKobylarz"`
* The `udl-list.json` will need a comma between the old `OSL_byEdawan` block and your new `UniBASIC_byJustinKobylarz` block

Until those are fixed, your submission cannot be merged.

**Comment by JustinKobylarz** [comment]
Oh Wow…  That did not go well!  😫

I think I got it all fixed up now…

Thanks for the insight!

Justin Kobylarz | Manager of IT & Owner
P (414) 352-9000 x179
D (414) 247-9833
C (414) 350-5677
PFlow Industries<https://www.pflow.com/> | Cartveyor®<https://www.cartveyor.com/> | PFlow International<https://www.pflow.com/>
***@***.***<https://qrco.de/bdkujN>
